ctype.md

```diff
@@ -66,11 +80,37 @@ Open it and paste the following:
     },
     "type": "object"
   },
-  "owner": "5EkPdXVsoJ8Vm8UTrs7c89ghJpWVZHGN7RGiQFSi6VC6RiX9",
-  "hash": "0x16fa158d730aa83f6339497fafa59d895c89c3eae24dc323532f71e235824152"
+  "owner": "5DD7fAZKSpgctg1ZQigAYuk3ypBtr2Q9RPKJBx5UpUwQw4vB",
+  "hash": "0xd8ad043d91d8fdbc382ee0ce33dc96af4ee62ab2d20f7980c49d3e577d80e5f5"
 }
 ```
 
 OK, now you've got all you need to create a claim: an identity and a CTYPE.
 
 Let's move on!
+
+### Side note: How to create a `CType` from a schema
+
+This step is **not part of the tutorial** and should be regarded as additional information on how to create a CType in the first place.
+Normally, you would start with a schema which does not have a defined `$id` yet:
+
+[comment]: <copy and paste 🚧 ctypeFromSchema_example from 2_ctypeFromSchema.ts>
+
+```javascript
+const Kilt = require('@kiltprotocol/sdk-js')
+
+const ctype = Kilt.CType.fromSchema({
+  $id: 'kilt:ctype:0x1',
```

Comment:
don't we auto-generate the id?

Reply:
You are right, it's unnecessary when using `fromSchema`.